app/src/androidTest/java/com/arctouch/codechallenge/util/MovieImageUrlBuilderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.arctouch.codechallenge.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MovieImageUrlBuilderTest {
```

Comment:
This was a mistake of mine, I should have done a Unit Test instead.

Reply:
Why did you choose to implement these tests as instrumented tests instead of unit tests?